Casks/c/codelite.rb

```diff
@@ -1,25 +1,57 @@
 cask "codelite" do
-  arch arm: "arm64", intel: "x86_64"
+  on_arm do
+    on_monterey do
+      arch arm: "arm64"
```

Comment:
```suggestion
      arch arm: "arm64"

```

Let's add a new line here. This should also be done for the following `arch` definitions.